src/main/cli/settings.c

```diff
@@ -935,6 +935,9 @@ const clivalue_t valueTable[] = {
     { "mixer_type",                 VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_MIXER_TYPE }, PG_MIXER_CONFIG, offsetof(mixerConfig_t, mixer_type) },
     { "crashflip_motor_percent",    VAR_UINT8 |  MASTER_VALUE,  .config.minmaxUnsigned = { 0, 100 }, PG_MIXER_CONFIG, offsetof(mixerConfig_t, crashflip_motor_percent) },
     { "crashflip_expo",    VAR_UINT8 |  MASTER_VALUE,  .config.minmaxUnsigned = { 0, 100 }, PG_MIXER_CONFIG, offsetof(mixerConfig_t, crashflip_expo) },
+#ifdef USE_DYN_LPF
+    { "dyn_lpf_update_rate",        VAR_UINT8 | MASTER_VALUE,  .config.minmaxUnsigned = { 10, 100 }, PG_MIXER_CONFIG, offsetof(mixerConfig_t, dyn_lpf_update_rate) },
```

Comment:
```suggestion
    { "dyn_lpf_update_rate_hz",        VAR_UINT8 | MASTER_VALUE,  .config.minmaxUnsigned = { 10, 100 }, PG_MIXER_CONFIG, offsetof(mixerConfig_t, dyn_lpf_update_rate) },
```

What unit does this setting have? If it's Hertz we should append "_hz" at the end.

Reply:
this setting has ms unit, 40 means 40*100 -> 4ms

Reply:
Okay, I see. Let me update my review